shopelectro/views/catalog.py

```diff
@@ -25,6 +25,13 @@ def get_products_count(request):
     return PRODUCTS_ON_PAGE_MOB if mobile_view else PRODUCTS_ON_PAGE_PC
 
 
+def get_paginated_page(objects, per_page, page_number):
+    try:
+        return Paginator(objects, per_page).page(page_number)
+    except:
```

Comment:
common except is not good practice

Reply:
@duker33 y, I forgot to catch here an `InvalidPage` error